testing/mock-server/server/server.go

```diff
@@ -105,6 +106,26 @@ func (s *GoopCSI) NVMfRemoteControllerStats(_ context2.Context, _ *pb.NVMfRemote
 	panic("implement me")
 }
 
+// ListNVMfRemoteNamespaces lists mock remote nvmf namespaces
+func (s *GoopCSI) ListNVMfRemoteNamespaces(context2.Context, *pb.ListNVMfRemoteNamespacesRequest) (*pb.ListNVMfRemoteNamespacesResponse, error) {
```

Comment:
I think we should postpone `NVMfRemoteNamespace` to a later PR
let's focus on PATH in this work ?

Reply:
It is just a stub with panic. Without this line, it won't compile